crates/bevy_reflect/src/serde/ser.rs

```diff
@@ -667,7 +665,7 @@ mod tests {
 
         let output = ron::ser::to_string_pretty(&serializer, config).unwrap();
         let expected = r#"{
-    "bevy_reflect::serde::ser::tests::should_serialize_option::OptionTest": (
+    "bevy_reflect::serde::ser::tests::OptionTest": (
```

Comment:
Shouldn't the function name be part of the type path when the type is declared inside it?

Reply:
Great question! Unfortunately, this is a limitation with `TypePath` since it relies on the Rust macro `module_path` to automatically generate the type path (and that particular macro does not take function names into account)